app/models/account/provider_methods.rb

```diff
@@ -35,7 +35,6 @@ module Account::ProviderMethods
     has_many :backend_api_metrics, through: :backend_apis, source: :metrics
     has_many :proxies, through: :services
     has_many :proxy_rules, through: :proxies
-    has_many :proxy_logs, foreign_key: :provider_id
```

Comment:
It is better first to remove the code then making the migration. Otherwise it will cause many errors on deploy